source/vscode/CHANGELOG.md

```diff
@@ -0,0 +1,182 @@
+- Fix panic in QIR generation when conditional branches use early return by [@swernli](https://github.com/swernli) in [#2388](https://github.com/microsoft/qsharp/pull/2388)
+- Ignore Dependency Errors when Getting Documentation by [@ScottCarda-MS](https://github.com/ScottCarda-MS) in [#2374](https://github.com/microsoft/qsharp/pull/2374)
+- Fix bug with Circuit CSS not being applied to notebooks by [@ScottCarda-MS](https://github.com/ScottCarda-MS) in [#2395](https://github.com/microsoft/qsharp/pull/2395)
+- Bug bashing fixes for qasm support by [@idavis](https://github.com/idavis) in [#2393](https://github.com/microsoft/qsharp/pull/2393)
+- Don't use the start of the target as the provider id by [@billti](https://github.com/billti) in [#2396](https://github.com/microsoft/qsharp/pull/2396)
+- Copilot tools for run, estimate, circuit by [@minestarks](https://github.com/minestarks) in [#2380](https://github.com/microsoft/qsharp/pull/2380)
+- Remove cell language override by [@idavis](https://github.com/idavis) in [#2399](https://github.com/microsoft/qsharp/pull/2399)
+- Sanitize messages by [@billti](https://github.com/billti) in [#2401](https://github.com/microsoft/qsharp/pull/2401)
+- Break on `fail` during debugging by [@swernli](https://github.com/swernli) in [#2400](https://github.com/microsoft/qsharp/pull/2400)
+- Added three OpenQASM samples by [@DmitryVasilevsky](https://github.com/DmitryVasilevsky) in [#2394](https://github.com/microsoft/qsharp/pull/2394)
+- Add explicit cast support by [@orpuente-MS](https://github.com/orpuente-MS) in [#2377](https://github.com/microsoft/qsharp/pull/2377)
+- Remove early return preventing panel from being made for empty circuit by [@ScottCarda-MS](https://github.com/ScottCarda-MS) in [#2402](https://github.com/microsoft/qsharp/pull/2402)
+- Add reset broadcast by [@idavis](https://github.com/idavis) in [#2404](https://github.com/microsoft/qsharp/pull/2404)
+- Limit inferred type sizes by [@swernli](https://github.com/swernli) in [#2382](https://github.com/microsoft/qsharp/pull/2382)
+- Add measurement broadcast by [@idavis](https://github.com/idavis) in [#2405](https://github.com/microsoft/qsharp/pull/2405)
+- bitstring processing is more efficient by [@idavis](https://github.com/idavis) in [#2408](https://github.com/microsoft/qsharp/pull/2408)
+- Restore fancy error reporting in Python by [@swernli](https://github.com/swernli) in [#2410](https://github.com/microsoft/qsharp/pull/2410)
+- OpenQASM Grover's algorithm sample by [@DmitryVasilevsky](https://github.com/DmitryVasilevsky) in [#2398](https://github.com/microsoft/qsharp/pull/2398)
+- OpenQASM Bernstein-Vazirani sample by [@DmitryVasilevsky](https://github.com/DmitryVasilevsky) in [#2403](https://github.com/microsoft/qsharp/pull/2403)
+- Users/merlinbot/1es pt auto baselining pr by [@idavis](https://github.com/idavis) in [#2371](https://github.com/microsoft/qsharp/pull/2371)
```

Comment:
We should trim this changelog to list things that are new that users would be relevant and interesting. Listing every commit re dependecy updates, pipline changes, etc. is too noisy.

Reply:
What is the vision for this view? I had originally imagined it would just have the info for the latest version, but I just put the changelogs for the last couple in here as a placeholder until we had that next version to talk about here. Are we wanting this to be a file that is continually added to version after version?

Reply:
I've trimmed off the items that seemed very implementation specific and tried to keep the user-facing items.